File: Class1/OX퀴즈.py
testCase = int(input())
answer = []
correctAnswer = [] # 

for _ in range(testCase):
    answer.append(input())

for result in answer:
    count = 0
    for c in result:
        if c == "O":
            count += 1
            correctAnswer.append(count)
        else:
            count = 0
            correctAnswer.append(count)
    print(sum(correctAnswer))
    correctAnswer = []

File: Algorithm/Algorithm/백준 구현/OX퀴즈.swift
//
//  main.swift
//  Algorithm
//
//  Created by 이건준 on 2023/09/22.
//

import Foundation

let testCase = Int(readLine()!)!

for _ in 0..<testCase {
  let result = readLine()!
  var count = 0
  var sum = 0
  result.forEach({ element in
    if element == "O" {
      count += 1
      sum += count
    }
    else {
      count = 0
    }
  })
  
  print(sum)
}

